### ISSUE TITLE
a way to pull docker image from host

### ISSUE THREAD

**Issue by jgallen23** [progrium/envy#29]
I have a base docker image for my dev setup and if I type `rebuild` it builds from that.  The problem is I regularly update that image, but rebuild isn't pulling latest, so I have to log into host machine and manually pull and then rebuild.  Any thoughts on making this easier?  I'd be happy to submit a pull request, but I wanted to discuss first in case something was planned already

**Comment by progrium** [comment]
So pulling latest on your base image that your Dockerfile references? That's tricky. I'd like to expose the host docker to admin users, and that would help in these situations ... except unless you wanted to do this for non-admin users. Hmm, I'd hate to slowly expose every Docker command.

**Comment by jgallen23** [comment]
Yes, pulling latest on the base image manually inside the host right now.  When I do a rebuild inside of an environment, it picks up the changes, but if I don't pull manually, it just rebuilds the previously pulled version.  I guess if I tagged my images as different versions, it would solve it, but right now docker hub is auto building and tagging as latest every time.

**Comment by progrium** [comment]
I was thinking of a "force" option for rebuild that would force rebuild
instead of using any cached layers. I suppose it could be worth doing a
pull on the FROM image in that case as well.

On Mon, Jun 29, 2015 at 10:53 AM, Greg Allen notifications@github.com
wrote:

> Yes, pulling latest on the base image manually inside the host right now.
> When I do a rebuild inside of an environment, it picks up the changes, but
> if I don't pull manually, it just rebuilds the previously pulled version. I
> guess if I tagged my images as different versions, it would solve it, but
> right now docker hub is auto building and tagging as latest every time.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/progrium/envy/issues/29#issuecomment-116774765.

## 

Jeff Lindsay
http://progrium.com
